src/main/java/es/upm/etsisi/views/CLI.java

```diff
@@ -80,4 +86,18 @@ private void displayHelp() {
             console.writeln(" - " + command.getName() + ": " + command.getDescription());
         }
     }
+
+    private void addDefaults() {
+        LinkedList<LinkedList<String>> defaultParameters = DefaultParameters.getDefaultParticipants();
+        Iterator<LinkedList<String>> iterator = defaultParameters.iterator();
+        while (iterator.hasNext()) {
+            LinkedList<String> currentParameters = iterator.next();
+            if (User.isUpmEmail(currentParameters.get(0))) {
+                this.controller.createPlayer(currentParameters.get(0), "default", currentParameters.get(1),
+                        currentParameters.get(2), DNI.generateDNI());
+            } else {
+                this.controller.createTeam(currentParameters.get(0), currentParameters.get(1));
+            }
+        }
+    }
```

Comment:
This method should be called setDefaults. It's easier to simply override the lists with valid generated ones

Reply:
You have to be sure that they are valid tho, keep that in mind

Reply:
Since DNI is randomly generated, we can have players with same name but different ID, as stated in #11
I left it as is since it is very unlikely that two players get same names and same DNI (which would be also incorrect with the current handling of ID, since they can get the same value, but different object pointer)

Will work on validation when #11's closed